eleventy-config.js

```diff
@@ -67,4 +81,12 @@ function getReadingListEntry(ep, readingList) {
   return { season, episode };
 }
 
+function getEpisodeEntry(rl, rlSeason, episodeList) {
```

Comment:
What does `rl` mean here? It seems like the episode you want. Naming the function `getEpisodeEntryFromList` might make things clearer, too.

Reply:
readling list.. the first parameter to the filter function what's piped into it, so I guess it'd actually be like readingListEntry

Reply:
Ah, maybe this function should be `getEpisodeForReadingListEntry`, then?